docs/source/contribute/gsoc_22.rst

```diff
@@ -0,0 +1,69 @@
+Google Summer of Code 2022
+============================================
+
+| **By**, `Omkar Phansopkar <https://github.com/OmkarPh>`_
+| **Project**: `Scancode-Workbench <https://github.com/nexB/scancode-workbench>`_
+| **Organization -** `AboutCode <https://www.aboutcode.org>`_
+
+Overview
+---------------
+ScanCode Workbench provides an advanced visual UI to help you quickly evaluate
+license and other notices identified by 
+`ScanCode <https://github.com/nexB/scancode-toolkit/>`_.
+ScanCode detects licenses, copyrights, packages
+and other interesting information in your code that can be visualised efficiently using workbench.
+
+The GSoC project's goal was to refactor ScanCode Workbench to a React + Typescript implementation and improve various sections of the application including Table view, file uploads, data sync across components, etc.
+Plus, the workbench wasn't updated for a long time, to support the latest scans from scancode-toolkit, hence the schema and UI had to be changed accordingly.
+
+Goals achieved
+-------------------
+
+* Refactor Workbench from vanilla javascript to React + Typescript implementation
+* Update Workbench to support the latest scancode-toolkit output format. (currently supports outputs from scancode-toolkit v31.1.1)
+* Improve User experience in Tableview, Home page, etc
+* Add convenience features like Drop files, History records, etc.
+* Add new sections for Scan Info, package info, etc
+
```

Comment:
You can also mention the automated release scripts, those were very important!

Reply:
yeah, sure